scheduler/conf.c

```diff
@@ -3525,6 +3525,7 @@ read_cupsd_conf(cups_file_t *fp)	/* I - File to read from */
     }
 
 	cupsdAddAlias(ServerAlias, value);
+  cupsdLogMessage(CUPSD_LOG_DEBUG, "Added ServerAlias %s", value);
```

Comment:
Please adjust the indentation as described in DEVELOPING.md, and add a dot at the end of the message. Thank you!